README.md

```diff
@@ -153,9 +153,16 @@ The html template file goes in the folder that you have defined in your TypoScri
         {record.renderedContent}
     </f:format.raw>
 </f:for>
+
+<f:for each="{children_202}" as="record">
+    {record.header} <br>
+    <f:format.raw>
+        {record.renderedContent}
+    </f:format.raw>
+</f:for>
```

Comment:
this gives no additional value to the example and blows up code unnecessary. Please remove it